client/app/components/routes/select-operator/form/UserOperator.tsx

```diff
@@ -23,6 +23,7 @@ async function getBusinessStructures() {
 export async function getUserOperatorFormData(id: number | string) {
   if (!id || isNaN(Number(id))) return {};
   return actionHandler(
+    // brianna
```

Comment:
Is this intentional?

Reply:
Definitely, I'm signing my work like an artiste